assets/js/status.js

```diff
@@ -8,28 +8,32 @@ const avatarProvider = "{{ site.avatar_provider }}";
 
 setTimeout(function() {
     $.getJSON(api + host, function(json) {
-        let count = 0;
-        let max = 0;
-        let motd = "";
-        let playersArray = [];
-
         const online = json.online;
 
         if (online) {
-            const players = json.players.online;
-            max = json.players.max;
-            motd = json.motd.clean[1];
-            const mapName = motd.replace('» ', '').replace(' «', '').replace(/.§./gi, '');
-            const mapUrl = imageProvider + mapName + "/map.png";
-            count += players;
-            playersArray = json.players.list;
-            fetchPlayerCount(count + "<small>/" + max + " players</small>");
-            fetchMapName(mapName);
+            const pgm = json.bukkit_extra.pgm;
+            const onlinePlayers = json.players.online;
+            const maxPlayers = json.players.max;
+            matchId = Object.keys(pgm)[0];
+            const currentMatch = pgm[matchId];
+            const currentMapName = currentMatch.map.name;
+            const currentMapObjective = currentMatch.map.objective;
+            const mapUrl = imageProvider + currentMapName + "/map.png";
+            const players = json.players.sample;
+            const tags = currentMatch.map.tags;
```

Comment:
`tags` may not exists when none are defined.

Reply:
I know `tags` may be `undefined` when none are defined. but even when no tags are defined, everything works fine. So I left it as it is.